src/search.js

```diff
@@ -0,0 +1,82 @@
+import { encode } from './encoding';
+import { getLinesLengthRanges } from './utils';
+
+export const searchIndexes = (rawKeywords, rawLog) => {
```

Comment:
I think there's a small bug with this function. When you try to search for something that matches from the beginning of the line, the highlighting works but attempting to filter the results doesn't work.

![Screen Shot 2019-05-16 at 10 01 57 AM](https://user-images.githubusercontent.com/3766511/57859921-e243d300-77c1-11e9-9920-03970cd81075.png)

In this example, instead of showing L17 & L21, it's showing L16 & L20
![Screen Shot 2019-05-16 at 10 02 12 AM](https://user-images.githubusercontent.com/3766511/57859920-e243d300-77c1-11e9-822e-6a3f076cee43.png)

Reply:
@ndiaz-candido Is this something you can take a look at? I would say that's probably the last thing stopping us from merging this PR.

Reply:
Hi @helfi92, sure, I'm in the middle of something but will take a look this weekend.

Reply:
That works. Thank you!

Reply:
@ndiaz-candido Any chance you had time to take a look? :-)

Reply:
Yes, I'm working on it :) I will try to get the fix as soon as possible. Has been a while since I opened the PR and had to refresh my memory with what I did.